src/graph/graph.h

```diff
@@ -39,11 +40,22 @@ typedef enum {
 	SYNC_POLICY_NOP,
 } MATRIX_POLICY;
 
+typedef enum {
+	GRAPH_CALLBACKS_UNDO,  // accumulate undo operations
+	GRAPH_CALLBACKS_NOP,   // discard undo operations
+} GRAPH_CALLBACKS_TYPE;
+
 // forward declaration of Graph struct
 typedef struct Graph Graph;
 // typedef for synchronization function pointer
 typedef void (*SyncMatrixFunc)(const Graph *, RG_Matrix);
 
+typedef enum {
+	ENTITY_UNKNOWN,
+	ENTITY_NODE,
+	ENTITY_EDGE,
+} EntityType;
```

Comment:
This seems like a duplicate of

```c
typedef enum GraphEntityType {
  1     GETYPE_UNKNOWN,
  2     GETYPE_NODE,
  3     GETYPE_EDGE
  4 } GraphEntityType;
```
within `graph_entity.h`
See if we can switch to `GraphEntityType`